tools/autodeployment/deploy_prod.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Shutdown
+cd ../telescope
+
+docker-compose -f docker-compose-staging.yml down
+
+docker system prune -af --volumes
+
+
+# Delete and clone
+cd ..
+
+rm -rf telescope
+
+git clone https://github.com/Seneca-CDOT/telescope.git --depth=1
+
+
+# Deploy
+cd telescope/
+
+cp env.staging .env
```

Comment:
`env.production`?

Is the only difference between this fine and `deploy_staging.sh` the name of the env file we use?  If so, let's rename this `deploy.sh` and have it take 1 argument: the filename of the env to use.

Reply:
Also, the passed argument should determine if the script uses `docker-compose-staging.yml` or `docker-compose-production.yml`